libtransmission/utils.c

```diff
@@ -2254,3 +2254,45 @@ void tr_net_init(void)
         initialized = true;
     }
 }
+
+/// mime-type
+
+#include "mime-types.c"
```

Comment:
I feel uneasy about including one C files into another (and not even listing this source file in CMakeLists.txt). If you're generating it, you could as well generate the corresponding header file to go along with it.

Reply:
fixed in fa8544d8f9b7f7e749ac0e4d6708aecbe348f1ce